src/services/HyperClient.ts

```diff
@@ -85,6 +85,27 @@ export default class HyperClient {
     return ip
   }
 
+  async getPodList (): Promise<Array<string>> {
+    if (this.config.noop) return ['']
+    const res = await axios.request({
+      socketPath: this.config.hyperSock,
+      method: 'get',
+      url: '/list?item=pod',
+      responseType: 'json'
+    })
+
+    const pods = res.data.podData.reduce((acc: Array<string>, curr: string) => {
+      const podDataArr = curr.split(':')
```

Comment:
What does `curr` represent? is it a URL or a tag or something else?

Reply:
Updated it to be hyperPod since its what comes back from the hyper list along with details of how its formatted.